src/main/java/org/jitsi/jigasi/SipGatewaySession.java

```diff
@@ -660,6 +676,38 @@ else if (sipCall != null)
         return mucDisplayName;
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void notifyForRecordingStatusChange(
+        JibriIq.RecordingMode mode, JibriIq.Status status)
+    {
+        // not a recording, ignore
+        if (!mode.equals(JibriIq.RecordingMode.FILE))
+        {
+            return;
+        }
+
+        // not a change, ignore
+        if (currentRecordingStatus.equals(status))
+        {
+            return;
+        }
+
+        if (JibriIq.Status.ON.equals(status))
+        {
+            // if call is still not established this will be ignored in
+            // injectSoundFile and nothing will be played
+            Util.injectSoundFile(this.call, REC_ON_SOUND);
```

Comment:
This is synchronous and takes about as long as the length of the audio file (i.e. seconds). Are we really OK blocking whatever thread fired the event for that long?